README.md

```diff
@@ -351,7 +355,7 @@ Here are `kinesis_firehose` specific configurations.
 Name of the delivery stream to put data.
 
 ### append_new_line
-Boolean. Default `true`. If it is enabled, the plugin add new line character (`\n`) to each serialized record.
+Boolean. Default `true`. If it is enabled, the plugin add new line character (`\n`) to each serialized record. At first, the overridden formatter in the plugin calls chomp and removes separator from the end of each record. Then, it appends `\n` to each record.
```

Comment:
It is unclear how it works with `chomp_record`. Could you add more details?

Reply:
Thanks, updated like [this](https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/172/files#diff-04c6e90faac2675aa89e2176d2eec7d8R357).